### PR TITLE
LPS-116119 Add Dictionary and Portal Directory to archived moduels up…

### PR DESCRIPTION
**Pull request by SamZiemer**
https://issues.liferay.com/browse/LPS-116119

### CODE REVIEW

**Comment by SamZiemer** [comment]
ci:test:stable

**Comment by SamZiemer** [comment]
ci:test:sf

**Comment by SamZiemer** [comment]
ci:test:relevant

**Comment by achaparro** [comment]
Just started reviewing :)

File: modules/apps/archived/archived-modules-upgrade/src/main/java/com/liferay/archived/modules/upgrade/internal/UpgradeDictionary.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Sam Ziemer
+ */
+public class UpgradeDictionary extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		LayoutTypeSettingsUtil.removePortletId(connection, "23");
```
Comment (achaparro) [inline modules/apps/archived/archived-modules-upgrade/src/main/java/com/liferay/archived/modules/upgrade/internal/UpgradeDictionary.java:26]: @SamZiemer, we should take into account that this portlet can have two different portletIds depending on the original version, 23 for 6.2 or below and com_liferay_dictionary_web_portlet_DictionaryPortlet for higher versions. Please take into account both cases. As a reference, see:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/archived/dictionary-web/src/main/java/com/liferay/dictionary/web/internal/upgrade/DictionaryWebUpgrade.java#L42

File: modules/apps/archived/archived-modules-upgrade/src/main/java/com/liferay/archived/modules/upgrade/internal/UpgradePortalDirectory.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Sam Ziemer
+ */
+public class UpgradePortalDirectory extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		LayoutTypeSettingsUtil.removePortletId(connection, "11");
```
Comment (achaparro) [inline modules/apps/archived/archived-modules-upgrade/src/main/java/com/liferay/archived/modules/upgrade/internal/UpgradePortalDirectory.java:26]: @SamZiemer for this case it is the same but we have to take into account that this plugin handles 4 portlets so we will need to verify 8 portletIds. See:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/archived/directory-web/src/main/java/com/liferay/directory/web/internal/upgrade/v1_0_0/UpgradePortletId.java#L28

File: modules/apps/archived/archived-modules-upgrade/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -1,7 +1,9 @@
 deprecated-modules-upgrade-configuration-name=Deprecated Modules Upgrade
 remove-chat-module-data=Remove chat module data during upgrade.
+remove-dictionary-module-data=Remove dictionary module data during upgrade.
```
Comment (achaparro) [inline modules/apps/archived/archived-modules-upgrade/src/main/resources/content/Language.properties:3]: @SamZiemer we need to execute "gw buildLang" to generate the translations for this keys. See this if you get any error:
https://liferay.slack.com/archives/C04D8HMEF/p1588276467112500

**Comment by achaparro** [comment]
I proceed to close the pull to add the changes requested

**Pull request closed without merging**